upstream/upstream_test.go

```diff
@@ -39,46 +27,15 @@ var _ = ginkgo.Describe("Upstream Push", ginkgo.Ordered, func() {
 		pushUpstream.setup(DefaultContext)
 		pushUpstream.StartServer()
 
-		Expect(pushUpstream.DB().Create(&models.Agent{ID: pushAgent.id, Name: pushAgent.name}).Error).To(BeNil())
-	})
-
-	ginkgo.It("should track insertion on the event_queue table", func() {
-		var events api.Events
-		err := pushAgent.DB().Where("name = ?", api.EventPushQueueCreate).Find(&events).Error
-		Expect(err).NotTo(HaveOccurred())
-		verifyPushQueue(events.ToPostQEvents(), pushAgent.dataset)
-	})
-
-	ginkgo.It("should track updates & deletes on the event_queue table", func() {
-		start := time.Now()
-
-		modifiedNewDummy := dummy.Logistics
-		modifiedNewDummy.ID = uuid.New()
+		DefaultContext.ClearCache()
+		context.SetLocalProperty("upstream.reconcile.pre-check", "false")
```

Comment:
We seem to have removed any verification and just assert no error ?

Reply:
The verification is still there